src/library.js

```diff
@@ -1856,14 +1856,21 @@ addToLibrary({
   },
 
 #if DYNCALLS || !WASM_BIGINT
+#if MINIMAL_RUNTIME
+  $dynCalls: '[]',
```

Comment:
This was `{}` before, and it does seem to be used more like a map than an array - why was this changed?

Reply:
Oops you are right... it should be {}